### PR TITLE
Add config flow for Utility Meter

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Updates documentation for the configuration flow that is added to the Utility Meter

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/68457
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by dgomes — DISMISSED** [review]
👍

File: source/_integrations/utility_meter.markdown [hunk 1]
```diff
@@ -23,9 +24,36 @@ Some utility providers have different tariffs according to time/resource availab
...
+    Enable this if you would like to treat the source as a net meter.
+    This will allow your counter to go both positive and negative.
+Delta values:
+  description: >
+    Enable this if the source values are delta values since the last reading instead of absolute values. When this option is enabled, each new value received will be added as-is to the utility meter instead of adding the _difference_ between the new value and previous value.
+{% endconfiguration_basic %}
+
+## YAML Configuration
+
+Alternatlively, this integration can be configured and set up manually via YAML
```
Comment (frenck) [inline source/_integrations/utility_meter.markdown]:
```suggestion
Alternatively, this integration can be configured and set up manually via YAML
```

File: source/_integrations/utility_meter.markdown [hunk 2]
```diff
@@ -23,9 +24,36 @@ Some utility providers have different tariffs according to time/resource availab
...
+Name:
+  description: The name the sensor should have. It can be changed again later.
+Input sensor:
+  description: The sensor entity providing utility readings (energy, water, gas, heating).
+Meter reset cycle:
+  description: How often to reset the counter.
+Meter reset offset:
+  description: >
+    Cycle reset occur at the beginning of the period. This option enables
+    the offsetting of these beginnings.
```
Comment (emontnemery) [inline source/_integrations/utility_meter.markdown]: We should mention config flows only supports an offset counted in days, and that yaml gives more freedom.
We should also mention that if the combination of "meter reset cycle" and "meter reset offset" is too limiting, yaml supports CRON patterns.

File: source/_integrations/utility_meter.markdown [hunk 3]
```diff
@@ -23,9 +24,45 @@ Some utility providers have different tariffs according to time/resource availab
...
+Meter reset cycle:
+  description: >
+    How often to reset the counter. If the offered reset cycles do not match
+    your use case, consider using the YAML configuration below, which allows
+    for creating CRON-style patterns.
+Meter reset offset:
+  description: >
+    Cycle reset occur at the beginning of the period. This option enables
+    the offsetting of these beginnings, counted in days. If you need a more
+    finegranded offset, consider using the YAML configuration below, which
```
Comment (emontnemery) [inline source/_integrations/utility_meter.markdown]:
```suggestion
    fine-grained offset, consider using the YAML configuration below, which
```

File: source/_integrations/utility_meter.markdown [hunk 4]
```diff
@@ -23,9 +24,45 @@ Some utility providers have different tariffs according to time/resource availab
...
+    allows for that.
+Supported tariffs:
+  description: >
+    A comma separated list of supported tariffs, leave empty if only a single 
+    tariff is needed.
```
Comment (emontnemery) [inline source/_integrations/utility_meter.markdown]:
```suggestion
    A list of supported tariffs, leave empty if only a single tariff is needed.
```

**Review by emontnemery — CHANGES_REQUESTED** [review]

**Pull request merged**